<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Azak_Roket_GUI</class>
 <widget class="QMainWindow" name="Azak_Roket_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Azak Roket Takımı Yer İstasyonu</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Masa/AZAK_LOGO.png</normaloff>../../Masa/AZAK_LOGO.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSensor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>161</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Sensör Verileri</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="textGyroX">
          <property name="text">
           <string>Gyro X :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textGyroY">
          <property name="text">
           <string>Gyro Y :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textGyroZ">
          <property name="text">
           <string>Gyro Z :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="gyroX">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="gyroY">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="gyroZ">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="textAngle">
        <property name="text">
         <string>Açı</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="angle">
        <property name="text">
         <string>???</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="textPress">
        <property name="text">
         <string>Yükseklik :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="press">
        <property name="text">
         <string>???</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="textAccX">
          <property name="text">
           <string>İvme X :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textAccY">
          <property name="text">
           <string>İvme Y :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textAccZ">
          <property name="text">
           <string>İvme Z :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="accX">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="accY">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="accZ">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxGPS">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>GPS Verileri</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="textLatitude">
          <property name="text">
           <string>Enlem :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLongitude">
          <property name="text">
           <string>Boylam :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="latitude">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="longitude">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxParachute">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Paraşüt Durum</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>194</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="textDragP">
          <property name="text">
           <string>Sürüklenme Paraşütü Durum :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textMainP">
          <property name="text">
           <string>Ana Paraşüt Durum : </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="dragP">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mainP">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxPort">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Port Bağlantı</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="textBaudRate">
          <property name="text">
           <string>BaudRate : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxBaudRate">
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="textComPort">
          <property name="text">
           <string>COMPort : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxComPort"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Bağlan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnectButton">
        <property name="text">
         <string>Bağlantıyı Kes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
